HouseRules_Essentials/Rules/BlockFriendlyFireRule.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System;
+    using System.Collections.Generic;
+    using Boardgame;
+    using Boardgame.BoardEntities;
+    using Boardgame.BoardEntities.Abilities;
+    using Boardgame.Data;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+    using UnityEngine;
+
+    public sealed class BlockFriendlyFireRule : Rule, IConfigWritable<bool>, IPatchable,
```

Comment:
nit: Prefer [passive voice for rule ](https://github.com/orendain/DemeoMods/blob/701de544367af323754366293c10caa85f5a9b5e/HouseRules_Essentials/Rules/SampleRule.cs#L12-L15)naming.